nautobot/extras/forms/forms.py

```diff
@@ -1286,6 +1286,13 @@ class JobResultFilterForm(BootstrapMixin, forms.Form):
         required=False,
         widget=StaticSelect2Multiple(),
     )
+    scheduled_job = DynamicModelMultipleChoiceField(
+        label="Scheduled Job",
+        queryset=ScheduledJob.objects.all(),
+        required=False,
+        to_field_name="name",
+        widget=APISelectMultiple(api_url="/api/extras/scheduled-jobs/"),
```

Comment:
An explicit `widget` shouldn't be needed here

Reply:
I'll remove it.  Honestly I'm not sure what it does, I was just referencing the other classes in the same file